apps/hubble-stats/app/layout.tsx

```diff
@@ -20,18 +24,72 @@ export default function RootLayout({
     setIsDarkMode('light');
   }, [setIsDarkMode]);
 
+  const items = [
+    {
+      name: 'Hubble',
+      isInternal: true,
+      href: '',
+      Icon: ContrastTwoLine,
+      subItems: [
+        {
+          name: 'Bridge',
+          isInternal: false,
+          href: 'https://app.webb.tools/#/bridge',
+        },
+        {
+          name: 'Explorer',
+          isInternal: true,
+          href: '',
+        },
+        {
+          name: 'Faucet',
+          isInternal: false,
+          href: 'https://develop--webb-faucet.netlify.app/',
```

Comment:
I'm curious as to why this directs to the development faucet rather than the main faucet page?

Reply:
Because it's incorrect :).